WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderConstants.java

```diff
@@ -47,6 +47,7 @@
     static final String ARG_IS_RELATED_POST = "is_related_post";
     static final String ARG_SEARCH_QUERY = "search_query";
     static final String ARG_VIDEO_URL = "video_url";
+    static final String ARG_IS_MAIN_READER = "is_main_reader";
```

Comment:
I found this name to be a little confusing. What about `ARG_IS_TOP_LEVEL`? or maybe it could be an enum instead of boolean?

Reply:
Naming changed here 37d20718591a2985cc62a79cbe0ef47751f6c09f . For now I would keep it simple with the boolean since we need to discriminate only between top level and rest of the world, let me know wdyt 😊